### ISSUE TITLE
Frame not works on macOS 12

### ISSUE THREAD

**Issue by Yang-Xijie** [Yang-Xijie/InputSourceSwitcher#14]
<img width="276" alt="image" src="https://user-images.githubusercontent.com/57952362/138833757-a7c6a641-7089-43f2-a105-6610d541f663.png">

**Comment by martinmts** [comment]
The AppleScript below does not work for Monterey. Is it a related issue?
```
	tell application "System Events"
		tell process "TextInputMenuAgent"
			click menu item layoutName of menu 1 of menu bar item 1 of menu bar 2
			click menu bar item 1 of menu bar 2
		end tell
	end tell
```

**Comment by Yang-Xijie** [comment]
This issue shows the UI problem. I have created a new issue for the functional problem you mentioned.

**Comment by Yang-Xijie** [comment]
Fixed in 73049b49756274d14b51c11f09b05efa3389e9fc
